picard/tagger.py

```diff
@@ -1464,6 +1468,9 @@ def process_picard_args():
                         help="do not load any plugins")
     parser.add_argument('--no-crash-dialog', action='store_true',
                         help="disable the crash dialog")
+    parser.add_argument('--debug-opts', action='store',
+                        default=None,
+                        help="Comma-separated list of debug options to enable: %s" % DebugOpt.opt_names())
```

Comment:
That's really minor, but all other options start with lowercase letters. We should be consistent

Reply:
Done in (rebased) https://github.com/metabrainz/picard/pull/2391/commits/61dbae164b69efe3c63de1446b67106f7079b7b6